reana_client/validation.py

```diff
@@ -392,6 +538,18 @@ def parse_command_params(step_value):
             ]
         return parse_command(str(step_value))
 
+    def save_publisher_definitions(
+        step, step_name, step_key, step_val, cmd_param_steps_mapping
```

Comment:
Minor: I think it would be more readable/explicit if the function just builds a dictionary of found parameters (so the function doesn't receive `cmd_param_steps_mapping`, but rather returns a new dict). Then when calling the function, we update the dictionary with the returned value:
```python
params_from_publisher = save_publisher_definitions(step, step_name, step_key, step_val)
cmd_param_steps_mapping.update(params_from_publisher)
```

Note: function `save_publisher_definitions` and variable `params_from_publisher` would have to be renamed to something more suitable.